app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] A user can have many posts, comments, and likes, please 🙏🏽 add the `foreign_key: 'author_id'` for all of them to have the correct relations between tables.

Reply:
fixed